a3p-integration/proposals/z:acceptance/test-lib/utils.js

```diff
@@ -93,3 +95,29 @@ export const makeTimerUtils = ({ setTimeout }) => {
     waitUntil,
   };
 };
+
+/**
+ * This function solves the limitation of getIncarnation when multiple Vats
+ * are returned for the provided vatName and does not return the incarnation
+ * of the desired Vat (e.g. zcf-mintHolder-USDC)
+ * @param {string} vatName
+ * @returns {Promise<number>}
+ */
+const getIncarnationFromDetails = async vatName => {
+  const matchingVats = await getDetailsMatchingVats(vatName);
+  const expectedVat = matchingVats.find(vat => vat.vatName === vatName);
```

Comment:
That checks the difference between 0 and at least 1. I was asking whether we need to worry about the difference between 1 and more than 1.  `find` will return the first instance. Do we need to worry about `getIncarnationFromDetails` returning a different vat than was intended because the `vatName` would have matched more than one vat?

Reply:
I don't think that should be a concern.
Assuming that getDetailsMatchingVats will return all vats that can be related with the provided vat name, and that vat names are unique, we can be confident that the strict equality operator will will one be true for the single instance that we intend to get.